src/domains/sign/io_sign_issuer_func.tf

```diff
@@ -0,0 +1,68 @@
+locals {
+  issuer_func_name      = format("%s-issuer-func", local.project)
+  issuer_func_container = "src"
+  issuer_func_package   = format("https://%sst.blob.core.windows.net/%s/%s.zip", replace(local.issuer_func_name, "-", ""), local.issuer_func_container, var.io_sign_issuer_func.version)
+}
+
+module "io_sign_issuer_func" {
+  source    = "git::https://github.com/pagopa/azurerm.git//function_app?ref=v3.3.0"
+  name      = local.issuer_func_name
+  subnet_id = module.io_sign_snet.id
+
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = var.io_sign_issuer_func.sku_tier
+    sku_size                     = var.io_sign_issuer_func.sku_size
+    maximum_elastic_worker_count = 1
+  }
+
+  os_type           = "linux"
+  always_on         = true
+  linux_fx_version  = "NODE|16"
+  health_check_path = "api/v1/sign/info"
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME                        = "node"
+    WEBSITE_NODE_DEFAULT_VERSION                    = "16.13.0"
+    WEBSITE_VNET_ROUTE_ALL                          = "1"
+    WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+    WEBSITE_DNS_SERVER                              = "168.63.129.16"
```

Comment:
where does this value come from?

Reply:
They come from our internal engineering standards. (https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/467435830/App+service#riepilogo-app-settings-obbligatori)

Other than being explained in Azure official doc:
* https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
* https://github.com/MicrosoftDocs/azure-docs/blob/29b43dee7ab917bfe6f372d7b8204375cad94fe7/includes/app-service-web-vnet-regional.md